docs/adr/0003-allowing-implicit-object-type.md

```diff
@@ -0,0 +1,47 @@
+# Allowing implicit object type
+
+## Status
+
+Proposed.
+
+## Context
+
+In real-world OpenAPI schema, we have discovered that people frequently omit the `type: object` from their object definitions. THIS IS NOT VALID Open API schema.
```

Comment:
Do we want to consider adding a strict mode? (Not in this PR. obvs.) We want to be able to support other people's iffy schemas, but I'd also like to know if our own schemas are wrong.

Reply:
This has now been superseded.